### ISSUE TITLE
Support for pytorch 1.13

### ISSUE THREAD

**Issue by benjats07** [Lightning-AI/lightning-flash#1502]
## 🚀 Feature
Support for pytorch 1.13 and MPS

### Motivation

I can't take advantage of MPS in macOS since is a recently added feature and lightning-flash didn't allow usage above torch 1.10

### Alternatives

I already make config changes to force the installation with newer torch and torchvision versions, but seems that 'nms' isn't available on torchvision.ops.boxes and icevision request that.

**Comment by benjats07** [comment]
I want to train models using MPS acceleration on MacOS but this capability is only available on lastest version of Pytorch. Especifically I want to follow this https://lightning-flash.readthedocs.io/en/latest/reference/object_detection.html

**Comment by Borda** [comment]
You can try to force upgrade Torch; I think all shall be working fine...
